### PR TITLE
generateSequance

### PR DESCRIPTION
**Pull request by ploWoj**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: generate-sequence
🏅 Granted 5 XP for @ploWoj

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ploWoj for on-time delivery

File: homework/generate-sequence/generateSequence.hpp [hunk 1]
```diff
@@ -2,6 +2,18 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+   
+    std::vector<int> vec{step};
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:6]: What if step is negate?

File: homework/generate-sequence/generateSequence.hpp [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+
+    if(count<=0)
+    {
+        return {};
+    }
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:11]: redundant

File: homework/generate-sequence/generateSequence.hpp [hunk 3]
```diff
@@ -2,6 +2,18 @@
...
+    //vec.reserve(count);
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:12]: Remove commented code

File: homework/generate-sequence/generateSequence.hpp [hunk 4]
```diff
@@ -2,6 +2,18 @@
...
+   for(size_t i{1}; i<count; i++)
+   {
+       vec.emplace(vec.end(),vec[i-1]+step);
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:15]: Add space between operations like '+' '-' '!=' '==' '%' etc...

Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:15]: Just use push_back, or if you resize vector already, use []

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**